ui/src/services/metric-service.ts

```diff
@@ -0,0 +1,60 @@
+import axios from "axios";
+import { SeriesOptionsType } from 'highcharts';
+const baseUrl = import.meta.env.VITE_BASE_URL;
```

Comment:
Question:  are we running into issues with this?  i.e., do we need different settings when running UI locally vs when being built?

If so, we've got the ENVIRONMENT key in `.env` which we can - and perhaps should? - leverage to simplify testing / deploying and remove a manual step.